src/stores/questionListStore.ts

```
@@ -0,0 +1,523 @@
+          const [removed] = questions.splice(fromIndex, 1);
+          questions.splice(toIndex, 0, removed);
+          
+          // 順序番号を更新して一括保存
+          const reorderedQuestions = questions.map((question, index) => ({
+            ...question,
+            order: index,
+          }));
+          
+          // 個別に更新（より効率的な方法は将来的に実装）
+          for (const question of reorderedQuestions) {
+            await dataStore.updateQuestion(listId, question.id, { order: question.order });
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)